src/ol_dbt/models/staging/mitxpro/_stg_mitxpro__models.yml

```
@@ -1525,6 +1525,8 @@ models:
   - name: cms_coursepage_duration
     description: str, a short description indicating how long the course takes to
       complete (e.g. '4 weeks')
+  - name: cms_coursepage_format
+    description: str, format of the course (Online, Other)
```

Will there be blank? if not, then I would suggest
```suggestion
    description: str, format of the program or course (Online, Other)
    tests:
    - not_null
    - accepted_values:
        values: ['Online', 'Other']
```